lib/hyrax/specs/shared_specs/hydra_works.rb

```diff
@@ -130,7 +130,6 @@
 
   it_behaves_like 'a Hyrax::Resource'
   it_behaves_like 'a model with core metadata'
-  it_behaves_like 'a model with basic metadata'
```

Comment:
i missed this on the first pass. just want to check that this is deliberate.

it seems right to remove it, at first glance, but also unrelated(?). is this a considered change?

Reply:
This was introduced by a rebase against master.  I just missed it when I created the new PR. It was brought in PR #4551 just a few days ago.  I will remove this change.

Reply:
Looking a little closer, this was GitHub's auto-rebase.  I didn't have this change in my local repo.  I'm rebasing locally and will push up again.

Reply:
This is resolved.  That was a good catch.